src/App.tsx

```diff
@@ -26,70 +26,127 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((a, b): number => {
+    switch (sortType) {
+      case SortType.ALPHABET:
```

Comment:
```suggestion
      case SortType.Alphabet:
```
Enum properties should be in PascalCase, not upper case